aries_cloudagent/protocols/present_proof/v2_0/routes.py

```diff
@@ -846,33 +878,64 @@ async def present_proof_send_presentation(request: web.BaseRequest):
         raise web.HTTPForbidden(reason=f"Connection {connection_id} not ready")
 
     pres_manager = V20PresManager(context.profile)
-    try:
-        indy_spec = body.get(V20PresFormat.Format.INDY.api)  # TODO: accommodate DIF
-        pres_ex_record, pres_message = await pres_manager.create_pres(
-            pres_ex_record,
-            {
-                "self_attested_attributes": indy_spec["self_attested_attributes"],
-                "requested_attributes": indy_spec["requested_attributes"],
-                "requested_predicates": indy_spec["requested_predicates"],
-            },
-            comment=body.get("comment"),
-            format_=fmt,
-        )
-        result = pres_ex_record.serialize()
-    except (
-        BaseModelError,
-        IndyHolderError,
-        LedgerError,
-        StorageError,
-        WalletNotFoundError,
-    ) as err:
-        return await internal_error(
-            err,
-            web.HTTPBadRequest,
-            pres_ex_record or conn_record,
-            outbound_handler,
-        )
 
+    if fmt == V20PresFormat.Format.INDY.api:
```

Comment:
There's quite some format specific code here. We need to define the schemas for the API, but besides that I think it would be nice to have all format specific stuff in the format handlers

Reply:
Removed all format-specific logic from `present_proof/v2_0/routes.py`. Updated `present_proof/v2_0/formats/indy/handler.py` and `present_proof/v2_0/formats/dif/handler.py` accordingly.